cluster-enroll-cobbler.sh

```diff
@@ -49,7 +49,7 @@ if [[ -f cluster.txt ]]; then
 		MATCH="$HOSTNAME"
 		ACTIONSTRING="adding $HOSTNAME.$DOMAIN ($IPADDR,$MACADDR) to cobbler with profile ${PROFILE}..."
 		echo $ACTIONSTRING | tee -a $TRANSCRIPT
-		sudo cobbler system add --name=$HOSTNAME --hostname=$HOSTNAME.$DOMAIN --profile=$PROFILE --ip-address=$IPADDR --mac=$MACADDR
+		sudo cobbler system add --name=$HOSTNAME --hostname=$HOSTNAME.$DOMAIN --profile=$PROFILE --ip-address=$IPADDR --interface=eth0 --mac=$MACADDR
```

Comment:
The bootstrap node's interface is defined in [`node["bcpc"]["bootstrap"]["pxe_interface"]`](https://github.com/bloomberg/chef-bach/blob/e9f40c07be07afabb2884a0078d7d3c1b31f3fbf/environments/Test-Laptop.json#L34); what is this interface used for (or why hardcoded to `eth0`)?